picard/formats/asf.py

```diff
@@ -195,7 +210,15 @@ def _load(self, filename):
                 if len(disc) > 1:
                     metadata["totaldiscs"] = disc[1]
                     values[0] = disc[0]
-            name = self.__RTRANS[name]
+            name_lower = name.lower()
+            if name in self.__RTRANS:
+                name = self.__RTRANS[name]
+            elif name_lower in self.__RTRANS_CI:
+                orig_name = name
+                name = self.__RTRANS_CI[name_lower]
+                self.__casemap[name] = orig_name
```

Comment:
May be a stupid question but is is possible to have the same tag with case variation more than once in original tags?

Reply:
Yes, but as they are treated case-insensitive they are treated as the same tag. In this case last casing wins. For replaygain the specs say there must be only one tag anyway.

Reply:
It _can_ happen, though it’s considered bad style.

Reply:
N.B.: For tag formats that support having "the same" tag in different case variations, loudgain currently checks for both (pure) lower- and uppercase variants and removes these before writing new tags. I’ll make a note to add handling for real case insensitivity in loudgain’s tag removal code.

Reply:
We can run into curious situations otherwise: Tagger A writes lowercase, tagger B writes uppercase, player A only reads lowercase, player B reads only uppercase. If values are different, the user might get surprising loudness variations and never know why. So I opted for removal of _both_ and rewrite in the case specified on the commandline (-L option).

Reply:
https://github.com/Moonbase59/loudgain/issues/10